### PR TITLE
feat: calculator

### PR DESCRIPTION
**Pull request by fromSian**

### CODE REVIEW

**Comment by fromSian** [comment]
why

**Comment by fromSian** [comment]
this is fine

File: code.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  constructor(value) {
+    this.value = value;
+  }
+  add(addV) {
+    this.value += addV;
+    return this;
+  }
+  subtract(subtractV) {
+    this.value -= subtractV;
+    return this;
```
Comment (sian-qian) [inline code.js]: I have select 3 lines

**Review by sian-qian — APPROVED** [review]
I wanna merge it

**Pull request merged**